scripts/test/bulkImportPerformance/waitForBulkImport.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Copyright 2022-2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPTS_DIR=$(cd "$(dirname "$0")" && cd ../.. && pwd)
+VERSION=$(cat "$SCRIPTS_DIR/templates/version.txt")
+JARS_DIR="$SCRIPTS_DIR/jars"
+GENERATED_DIR="$SCRIPTS_DIR/generated"
+
+SYSTEM_TEST_JAR="$JARS_DIR/system-test-$VERSION-utility.jar"
+
+INSTANCE_PROPERTIES=${GENERATED_DIR}/instance.properties
+INSTANCE_ID=$(grep -F sleeper.id "${INSTANCE_PROPERTIES}" | cut -d'=' -f2)
+
+source "$SCRIPTS_DIR/functions/timeUtils.sh"
+START_TIME=$(record_time)
+
+echo "-------------------------------------------------------------------------------"
+echo "Waiting for EMR clusters"
+echo "-------------------------------------------------------------------------------"
+java -cp "${SYSTEM_TEST_JAR}" \
+sleeper.systemtest.bulkimport.WaitForEMRClusters "${INSTANCE_ID}"
```

Comment:
There's a case where the bulk import jobs have been submitted but the EMR clusters haven't yet been created when we get to this point. We're waiting for the EMR clusters to finish, but we also need to wait for them to be created.